snuba/clusters/cluster.py

```diff
@@ -251,6 +251,28 @@ def get_batch_writer(
             buffer_size=buffer_size,
         )
 
+    def get_values_batch_writer(
+        self,
+        metrics: MetricsBackend,
+        insert_statement: InsertStatement,
+        encoding: Optional[str],
+        options: ClickhouseWriterOptions,
+        chunk_size: Optional[int],
+        buffer_size: int,
+    ) -> BatchWriter[bytes]:
+        return HTTPBatchWriter(
+            host=self.__query_node.host_name,
+            port=self.__http_port,
+            user=self.__user,
+            password=self.__password,
+            metrics=metrics,
+            statement=insert_statement.with_database(self.__database),
+            encoding=encoding,
+            options=options,
+            chunk_size=chunk_size,
+            buffer_size=buffer_size,
+        )
+
```

Comment:
This seems identical to the one above ?

Reply:
It's true that this is equivalent though that assumes `JSONRow=bytes` (in terms of return typing). I think I went down a path that didn't yield anything here by separating it out.